emulators/oberon-risc-emu/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                oberon-risc-emu
+version             20200818
+categories          emulators
+platforms           darwin
+maintainers         {@jflude hotmail.com:justin_flude} openmaintainer
+description         Project Oberon RISC system emulator
+long_description    Project Oberon 2013, by Niklaus Wirth and Jürg Gutknecht, \
+    is a lean, easily understandable  desktop computer. This emulation, \
+    risc, includes a disk image for the full system (NB. make a writable copy)
+homepage            http://www.projectoberon.net/
+license             Permissive
+
+fetch.type          git
+git.url             https://github.com/pdewacht/oberon-risc-emu
```

Comment:
You may be happier using the github portgroup.

Reply:
Done (it's slightly more fiddly that way as you need checksums)